### DISCUSSION TITLE
MLB Network

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#344] [Show and tell]
### Channel Name

MLB Network 

### Channel ID

MLBNetwork.us

### Stream URL

https://tve-live.mlb.com/1646667328_91f7324493a7b667ee877fb03e941e6a9de90ea189d392e5ae1cdd432f12d2b9/TVE/sYolL-AHq87-33XJF-J2VV8-jEsrL/mlbn/live/a-net.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by BellezaEmporium** [comment]
Not quite sure if this one is DMCAd or not.

**Comment by s66jones** [comment]
https://ustvgo.tv/mlb-network/

**Comment by freearhey** [comment]
https://tve-live.mlb.com/1646667328_91f7324493a7b667ee877fb03e941e6a9de90ea189d392e5ae1cdd432f12d2b9/TVE/sYolL-AHq87-33XJF-J2VV8-jEsrL/mlbn/live/a-net.m3u8 (offline)
<img width="836" alt="image" src="https://user-images.githubusercontent.com/7253922/232225709-c16b355b-2284-4b5f-8d4f-e32251e4d6be.png">
